validations/src/tests.rs

```diff
@@ -87,19 +87,18 @@ fn sign_t3<H: Hashable>(tx: &H) -> KeyedSignature {
 }
 
 // Counter used to prevent creating two transactions with the same hash
-static TX_COUNTER: AtomicU32 = AtomicU32::new(0);
+// static TX_COUNTER: AtomicU32 = AtomicU32::new(0);
```

Comment:
here too

Reply:
Oh yeah, now I remember. The epoch field is used to ensure that every `MintTransaction` has a unique hash, because otherwise when one node mines two empty blocks, the hash of the mint transaction will be the same in this two blocks, as it only depends on pkh and value.